server/db/mongodb/adapter.go

```diff
@@ -138,6 +142,10 @@ func (a *adapter) Open(jsonconfig json.RawMessage) error {
 		a.maxResults = defaultMaxResults
 	}
 
+	if a.maxMessageResults <= 0 {
+		a.maxMessageResults = defaultMaxMessageResults
```

Comment:
`defaultMaxMessageResults` is not defined anywhere.

Reply:
Done.